examples/fifaText/README.md

```diff
@@ -0,0 +1,9 @@
+# Twitter handle classifier
+this example is used by [@soypete](https://github.com/Soypete) at Gopherpalooza 2019. The talking points were as follows. 
+
+- Understand your data
+- Feature engineering
+- Train and Test the model
+- Deploy and Maintain
+
+The slides can be found [here]()
```

Comment:
I guess this is not an error and the link is fulfilled once the slides are online.